glancesync/glancesync_serversfacade.py

```diff
@@ -118,7 +183,23 @@ def delete_image(self, regionobj, id, confirm=True):
         :param confirm: ask for confirmation
         :return: true if image was deleted, false if it was canceled by user
         """
-        return _delete_image(regionobj, id, confirm)
+        client = self._get_glanceclient(regionobj.region)
+        if confirm:
+            confirm = raw_input('Delete image {0}? [y/N]'.format(id))
```

Comment:
I don't see the point. This is a parameter and it is already fully documented. 

The glancesync code by the way already invokes this method with confirm=False. The only code that invokes this method with confirm=True is in examples folder.

The default value is True only for promoting using this method carefully. Glancesync only uses it after overriding an image with other with a different checksum.